File: rtiow/Go/main02.go
package main

import ("fmt")

func main() {
	var nx = 200
	var ny = 100
	fmt.Printf("P3\n")
	fmt.Printf("%d %d\n", nx, ny)
	fmt.Printf("%d\n", 255)
	for j := ny-1; j >= 0; j-- {
		for i := 0; i < nx; i++ {
			r := float32(i) / float32(nx)
			g := float32(j) / float32(ny)
			b := float32(0.2)
			ir := int(255.99 * r)
			ig := int(255.99 * g)
			ib := int(255.99 * b)
			fmt.Printf("%d %d %d\n", ir, ig, ib)
		}
	}
}

File: rtiow/Nim/main02.nim
import strformat
var nx = 200
var ny = 100
echo("P3")
echo(fmt"{nx} {ny}")
echo(255)
for j in countdown(ny - 1, 0):
 for i in countup(0, nx - 1):
  var r = i / nx
  var g = j / ny
  var b = 0.2
  var ir = int(255.99 * r)
  var ig = int(255.99 * g)
  var ib = int(255.99 * b)
  echo(fmt"{ir} {ig} {ib}")
